### PR TITLE
Create build matrix with different nodeJS versions

### PR DESCRIPTION
**Pull request by webmaster128**
I suspect we have bugs on master that are not detected in nodejs8. Let's see

### CODE REVIEW

**Comment by apanizo** [comment]
Travis automatic deployment:
https://pr330-365__iov.surge.sh

Storybook book automatic deployment:
 https://storybook_pr330-365__iov.surge.sh

**Comment by apanizo** [comment]
Travis automatic deployment:
https://pr330-65__iov.surge.sh

Storybook book automatic deployment:
 https://storybook_pr330-65__iov.surge.sh

**Comment by apanizo** [comment]
Travis automatic deployment:
https://pr330-287__iov.surge.sh

Storybook book automatic deployment:
 https://storybook_pr330-287__iov.surge.sh

**Comment by apanizo** [comment]
Travis automatic deployment:
https://pr330-210__iov.surge.sh

Storybook book automatic deployment:
 https://storybook_pr330-210__iov.surge.sh

File: webpack/webpack.common.config.js [hunk 1]
```diff
@@ -16,8 +16,6 @@ fs.writeFileSync(configFile, JSON.stringify(config));
 // style files regexes
 const cssRegex = /\.css$/;
 const cssModuleRegex = /\.module\.css$/;
-const sassRegex = /\.(scss|sass)$/;
-const sassModuleRegex = /\.module\.(scss|sass)$/;
```
Comment (apanizo) [inline webpack/webpack.common.config.js]: 🤔 why?

File: scripts/test_start.sh [hunk 2]
```diff
@@ -10,33 +10,36 @@ command -v shellcheck > /dev/null && shellcheck "$0"
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```
Comment (apanizo) [inline scripts/test_start.sh]: I am lost....

File: webpack/webpack.common.config.js (hunk 1 above)
Comment (webmaster128) [reply webpack/webpack.common.config.js]: Both are unused variables. I found then when I searched the project for sass and it turned out we have nothing to do with sass

File: scripts/test_start.sh (hunk 2 above)
Comment (webmaster128) [reply scripts/test_start.sh]: The important change in test_start.sh is that sometimes faucet started too early, at a point where the blockchain was not ready. I moved the faucet starts down to givve the blockchains some time to get warm. A few seconds are enough.

File: webpack/webpack.common.config.js (hunk 1 above)
Comment (apanizo) [reply webpack/webpack.common.config.js]: Ey! But we do have `scss` files, right?

File: scripts/test_start.sh (hunk 2 above)
Comment (apanizo) [reply scripts/test_start.sh]: Perfect.

File: webpack/webpack.common.config.js (hunk 1 above)
Comment (webmaster128) [reply webpack/webpack.common.config.js]: Are they processed by the style-loader and or css-loader here https://github.com/iov-one/wallet-demo/blob/v0.12.2/webpack/webpack.common.config.js#L80-L101?

Comment (apanizo) [reply webpack/webpack.common.config.js]: Correct!

**Comment by apanizo** [comment]
Travis automatic deployment:
https://pr330-109__iov.surge.sh

Storybook book automatic deployment:
 https://storybook_pr330-109__iov.surge.sh

**Pull request merged**